snippets/page-metadata.liquid

```diff
@@ -1,8 +1,4 @@
-{% if page_title == blank %}
-  <title>{{ shop.name }}</title>
-  <meta property="og:title" content="{{ shop.name }}">
-  <meta name="twitter:title" content="{{ shop.name }}">
-{% else %}
+{% if page_title != blank %}
```

Comment:
While it is technically possible to omit the <title> tag in the HTML <head> section, but as far as I know it is not recommended to do for some reasons

Reply:
i can remove the `if` clause. The page title is defined on the global drop, so it should be present all the times

https://github.com/booqable/booqable/blob/63d3d9f425ea6a7b81a9f9513d75502eb755addc/app/lib/shop_next/drops/global_drop.rb#L117

what you think?

Reply:
hum but thats the thing. It will always have a page title, that fallbacks to Company name. 

Are you able to provide me the steps where title will show empty? Sorry, but i'm still navigating and learning the codebase :D